### PR TITLE
tabbar: move proptypes to another file

### PR DESCRIPTION
**Pull request by darakeon**

### CODE REVIEW

File: webapp/src/components/TabBar/style.css [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 .TabList {
   margin: var(--tabbar-tablist-padding-fix);
+  color: var(--tabbar-tablist-padding-fix);
```
Comment (felquis) [inline webapp/src/components/TabBar/style.css]: ?

Comment (darakeon) [reply webapp/src/components/TabBar/style.css]: Same feeling here...

**Comment by darakeon** [comment]
Done by cherry-pick

**Comment by darakeon** [comment]
Done by cherry-pick

**Pull request closed without merging**